### ISSUE TITLE
Provide a Makefile?

### ISSUE THREAD

**Issue by nickludlam** [OtherCrashOverride/XU4VideoCube#1]
Great example code, thank you. Is there any chance you could provide a Makefile?

**Comment by OtherCrashOverride** [comment]
Yup, I totally forgot about that one.  Its was developed as a Code::Blocks project and the IDE workspace file (XU4VideoCube.cbp) is there to compile and run it.

To manually compile (or create a Makefile) simply compile everything with a *.cpp extension and link it all together with libmali.so and libX11.so.

The only define used is "DEBUG=1" for a debug build that enables checking OpenGL error status.

Pull requests welcome for this.
